bigtable/docs/snippets.py

```diff
@@ -403,7 +403,18 @@ def test_bigtable_create_table():
     table = instance.table("table_my")
     # Define the GC policy to retain only the most recent 2 versions.
     max_versions_rule = column_family.MaxVersionsGCRule(2)
-    table.create(column_families={"cf1": max_versions_rule})
+
+    RETRIES = 4
+    # Retry if deadline exceed error.
+    for retry in range(RETRIES):
+        try:
+            table.create(column_families={"cf1": max_versions_rule})
+            break
+        except Exception as e:
+            if retry == RETRIES - 1:
+                raise e
+            else:
+                time.sleep(2 ** (retry - 1))
```

Comment:
Please use `test_utils.retry.RetryErrors` with a specific exception here, e.g.:

```python
from google.api_core.exceptions import DeadlineExceeded

retry_504 = RetryExceptions(DeadlineExceeded)
...
        retry_504(table_create)(...)
```

Reply:
Retry DeadlineExceeded exception is good idea.
But I have kept a simple retry since this will be included in snippet, and will be easy for user to understand w/o worrying about our retry imports.

If you still feel we should include explicit exceptions in snippets, I'll do it.

Reply:
I missed that this was inside the snippet:  there, rather than using the `RetryExceptions` class from `test_utils`, we should illustrate using the one from `google.api_core.retry`, e.g.:

```python
from google.api_core import exceptions
from google.api_core import retry

...

# Could include other retriable exception types
predicate_504 = retry.if_exception_type(exceptions.DeadlineExceeded)

# Could configure deadline, etc.
retry_504 = retry.Retry(predicate_504)

retry_504(table_create)(...)
```

Reply:
Done.